Remove mentions of Wanderers in news before you learn their language (#6765)
endless-sky/endless-sky: data/wanderer/wanderer news.txt

# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

news "wanderer civilians, no dialog 1, pre language"
	to show

> 		not "language: Wanderer"

Reviewer: Sorry, my comment was not fully clear. What I meant is that we have a set of messages that is "always shown" and that don't refer to the Wanderers by name, and another set of messages that is "only shown after learning their name" and that does refer to the Wanderers by name.

	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Alien"
	message
		word
			"Many buildings near the spaceport have plants growing on their roofs. An indignant squawk draws your attention to an alien as they rapidly flap away from an automated watering system dousing them where they stood."

> 			"Outside the spaceport, you see an alien performing maintenance on some sort of irrigation system. Once they get it working, it begins to water a field of plants."
> 			"An alien dockworker unloads complex scientific equipment from a ship. More surprising, however, is the fact that they unload several bags of fertilizer from it as well."

Reviewer: I'm not too happy to have all the entries copied, with just the word Wanderer replaced by alien.

Can we make some of the messages more generic, like for example those two and have only the generic ones shown before learning the Wanderers name? (And both the generic ones and the Wanderer specific ones shown after learning their name?)

```suggestion
			"Outside the spaceport, you see maintenance being performed on some sort of irrigation system. Once they get it working, it begins to water a field of plants."
			"A dockworker unloads complex scientific equipment from a ship. More surprising, however, is the fact that they unload several bags of fertilizer from it as well."
```
